conduct.md

```diff
@@ -132,7 +132,9 @@ TODO: Write an introductory paragraph.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are violations of the honor code and campus policies. Depending upon the infraction one commits and past infractions, different penalties are applied to the student.
+
+Infractions are split into different categories and each infraction category adds to a point total when committed. For example, if you commit three Level 1 infractions, that is the same as committing one Level 2 infraction. The higher the total infraction points you have, the more severe of a penalty you may receive.
```

Comment:
Exactly what constitutes a level 1, level 2, or level 3 infraction should be clearly defined in this conduct guide. It seemed that @Yanqiao4396 was defining infractions in his PR #7. However, what he has written is not consistent with what is written here. The definitions for infraction levels should be discussed as a team.